app/assets/stylesheets/avalon.scss

```diff
@@ -576,6 +601,10 @@ div.status-detail {
     span.tool_label {
       font-weight: bold;
     }
+
+    div.transcript_file_view {
+      display: flex;
+    }
```

Comment:
It doesn't look like this is used.  Can it be removed?

Reply:
Yes, good catch :+1: